pkg/api/server/server.go

```diff
@@ -205,20 +223,20 @@ func (s *APIServer) Serve() error {
 
 	go func() {
 		<-s.idleTracker.Done()
-		logrus.Debugf("API Server idle for %s", s.idleTracker.Duration.Round(time.Second).String())
+		logrus.Debug("API service shutting down idle for " + s.idleTracker.Duration.Round(time.Second).String())
```

Comment:
Maybe a colon between "shutting down" and "idle"? Or reword to "shutting down after idling for"?